packages/chevrotain/tsconfig.es6.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "moduleResolution": "node",
+        "module": "esnext",
+        "target": "es6",
+        "outDir": "es6"
```

Comment:
can we use an outFile to be friendly for browser usage?
I think vuejs does it.

Reply:
I think, for browser users, if they want to just include one single file with <script> tag in html, they can just use the bundled file (https://unpkg.com/chevrotain/lib/chevrotain.js
), which is not influenced by this es6 build.

Only those who are using bundlers to develop for browsers will need this es6 build and they won't need one `outFile`.

Reply:
In #976 The request was specifically for an esm bundle for browser use.
The use case was using Chevrotain inside three.js library to parse some  3D format.
So perhaps if we build a:
- Single file
- esm compatiable.
- ES5 syntax

Build we would target the lowest common denominator for those requiring esm moduls builds?
Could such a single file ESM bundle be used directly both in the browser and in node.js (tree shacking?)

Reply:
Sorry for missing this issue. I don't think the developer must use a single esm file, can you ask him again for clarity?

Reply:
> Could such a single file ESM bundle be used directly both in the browser and in node.js (tree shacking?)

Yes, I think it can!

So do we only need one single bundled file in esm instead of separated sub-modules?